examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -697,7 +697,7 @@ static void oneIndoorInit(void) {
   ledBarEnabledUpdate();
 
   /** Show message init sensor */
-  oledDisplay.setText("Sensor", "initializing...", "");
+  oledDisplay.setText("Monitor", "initializing...", "");
```

Comment:
Not related to the PR, changed this because we then initialize multiple sensors, so we can also say that we initialize the entire monitor.